definitions/procedures/foreman_docker/remove_foreman_docker.rb

```diff
@@ -6,11 +6,12 @@ class RemoveForemanDocker < ForemanMaintain::Procedure
     end
 
     def docker_package
-      'tfm-rubygem-foreman_docker'
+      find_package(foreman_plugin_name('foreman_docker'))
```

Comment:
Doesn't this return `nil` if it's not found? There's also already a `return unless` on line 13 so I'd think this is sufficient:
```suggestion
      foreman_plugin_name('foreman_docker')
```

Reply:
Good catch